### DISCUSSION TITLE
Filament and Blueprint?

### DISCUSSION THREAD

**Discussion by silicahd** [filamentphp/filament#849] [General]
Anybody has any idea id Filament works with Blueprint?

**Comment by internetbug256** [comment]
Are you talking about laravel-shift/blueprint and filamentPHP?
I just found your post while looking for this. I am quite interested in using Bluprint to speed up the programming of Filament scripts.

**Comment by MattMello88** [comment]
this is a very good ideal to created, i am quite interested too
